src/services/login/useExtensionLogin.ts

```diff
@@ -70,11 +70,12 @@ export const useExtensionLogin = ({
         ...(signature ? { signature } : {}),
         ...(token ? { loginToken: token } : {})
       });
-
-      window.location.href = `${url.pathname}?${nextUrlParams}`;
```

Comment:
dispatch is not finishing and it already redirected to the new location.
If I added a setTimeout and changed the order things worked for us.
hope it helps